e2e/tests/interchain_accounts/params_test.go

```diff
@@ -111,6 +112,7 @@ func (s *InterchainAccountsParamsTestSuite) TestControllerEnabledParam() {
 	})
 }
 
+// compatibility:TestHostEnabledParam:from_versions: v9.0.0,v8.4.0,v7.5.0
```

Comment:
this syntax is arbitrary, we can change this to whatever we like. I figured it is a straightforward enough way of specifying custom data about specific tests.

Reply:
Should we document the syntax somewhere?

Reply:
yep I will add a README file to go along with the script!